packages/pigeon/platform_tests/test_plugin/windows/test_plugin.cpp

```diff
@@ -65,7 +65,15 @@ std::optional<FlutterError> TestPlugin::ThrowErrorFromVoid() {
   return FlutterError("An error");
 }
 
-ErrorOr<int64_t> TestPlugin::EchoInt(int64_t an_int) { return an_int; }
+ErrorOr<std::optional<flutter::EncodableValue>> TestPlugin::ThrowFlutterError(){
+    return FlutterError("code", "message", EncodableValue("details"))}
+
+std::optional<FlutterError> TestPlugin::ThrowFlutterErrorFromVoid(){
+    return FlutterError("code", "message", EncodableValue("details"))}
```

Comment:
The formatting on these return lines is incorrect, you'll need a semi-colon after each return (and the `}` should be on it's own line)